grafana-plugin/src/pages/escalation-chains/EscalationChains.tsx

```diff
@@ -161,17 +162,19 @@ class EscalationChainsPage extends React.Component<EscalationChainsPageProps, Es
               {!searchResult || searchResult.length ? (
                 <div className={cx('escalations')}>
                   <div className={cx('left-column')}>
-                    <WithPermissionControlTooltip userAction={UserActions.IntegrationsWrite}>
-                      <Button
-                        onClick={() => {
-                          this.setState({ showCreateEscalationChainModal: true });
-                        }}
-                        icon="plus"
-                        className={cx('new-escalation-chain')}
-                      >
-                        New Escalation Chain
-                      </Button>
-                    </WithPermissionControlTooltip>
+                    {!loading && (
```

Comment:
did not get why do we hide Create Escalation Chain button until escalation chains loaded

Reply:
this was a quirky scenario in the UI that I discussed w/ @teodosii. Basically we show two different "New Escalation Chain" buttons:
- one while the HTTP request to fetch escalation chains is loading
- one if there are no escalation chains from the response
This made it really difficult to reliably target one of these buttons